lib/presentation/ui/utils/mapbox_copyright.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:vost/presentation/utils/misc.dart';
+
+/*
+ * Created by Gonçalo Lima (https://github.com/lima21) for Fogos.pt
```

Comment:
What do you think of having this kind of things in a section on contribution page?

Reply:
It will be difficult to attribute to each person, since we currently have a list of contributors.

Don't you think this might be a good idea to leave in the code?

Reply:
That's ok by me, I was just wondering what could be the best place for that. We'll leave it as it is for now.

Reply:
👍